pkg/database/database.go

```diff
@@ -155,7 +155,10 @@ func (d *Database) CreateFixturesSync() error {
 	}
 
 	output := strings.Join(statements, ";\n")
-	output = fmt.Sprintf("/* Auto generated file. Do not edit by hand. This file was generated by SchemaHero. */\n\n %s;\n\n", output)
+	if output != "" {
+		output = output + ";\n"
+	}
+	output = fmt.Sprintf("/* Auto generated file. Do not edit by hand. This file was generated by SchemaHero. */\n\n%s\n", output)
```

Comment:
this change just fixes a scenario when all tables are dropped.  it would just return a migration like:
```
 ;
```

This also removes a leading space before the `output`.  Assuming that was not needed for any reason?